### PR TITLE
[Infrastructure] Add a docker guide with AWS and Heroku examples

### PR DESCRIPTION
**Pull request by gussiciliano**
### SUMMARY
- Add a Docker guide
- Add deploy in AWS using Docker
- Add deploy in Heroku using Docker

### CODE REVIEW

File: infrastructure/docs/docker-guide.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+## Docker guide
```
Comment (mdesanti) [inline infrastructure/docs/docker-guide.md]: @gussiciliano great job here!

I would separate this into 3 files. One for docer common configuration, one for AWS and one for Heroku. What do you think?

Comment (gussiciliano) [reply infrastructure/docs/docker-guide.md]: yes :) it is a good idea

I want to thank @truccomarcos  for the research he did on this topic :D

**Comment by gussiciliano** [comment]
@mdesanti I already made the changes :D
@ziflit @Martum please, check if I can put more and better information, especially in infrastructure/docs/docker/README.md :D

File: infrastructure/docs/docker/docs/docker-aws.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ```
+
+2. Configure your access credentials, creating a file in your root folder `~/.aws/credentials`. Make sure it has the following format:
+    ```bash
+        [profile_name]
+        aws_access_key_id = your_access_key
+        aws_secret_access_key = your_access_key_secret
+    ```
+
+3. Create a Dockerrun.aws.json file in the root directory of your application. It should look like this:
```
Comment (ziflit) [inline infrastructure/docs/docker/docs/docker-aws.md]: (This link)[https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_image.html#create_deploy_docker_image_dockerrun] provides some more info

**Comment by ziflit** [comment]
Also, careful in https://github.com/Wolox/netcore-bootstrap/blob/master/NetCoreBootstrap/Dockerfile where you left your project name hardcoded into the Dockerfile. It's not that it's wrong but could be misleading

File: infrastructure/docs/docker/docs/docker-aws.md (hunk 2 above)
Comment (gussiciliano) [reply infrastructure/docs/docker/docs/docker-aws.md]: Great! I'm going to add this link

File: infrastructure/docs/docker/docs/docker-aws.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#### Deploy to AWS (Building Docker image on AWS)
+
+AWS has the possibility of directly building a Docker image, allowing us to just push our code (with a [valid Dockerfile](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_image.html#create_deploy_docker_image_dockerfile)). 
+
+1. Download the [Elastic Beanstalk Client](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-install.html)
```
[the remark this answers is not in the document]
Comment (gussiciliano) [reply infrastructure/docs/docker/docs/docker-aws.md]: Ok :)
I'm going to change this to referer to https://github.com/Wolox/tech-guides/blob/master/infrastructure/docs/aws/docs/aws-eb-deploy.md#installing-the-tools :D

**Comment by gussiciliano** [comment]
@ziflit thanks for your comments, I add a new commit on this PR and also I changed the Dockerfile in https://github.com/Wolox/netcore-bootstrap/blob/master/NetCoreBootstrap/Dockerfile

**Pull request merged**